### DISCUSSION TITLE
Pivot data on a custom page table

### DISCUSSION THREAD

**Discussion by ramiropiro** [filamentphp/filament#6991] [General]
hello, i need to arrange data from a pivot on a custom page but all the records from a specific user appears in the same row, how can i make them work like a relation manager?
![image](https://github.com/filamentphp/filament/assets/111913304/186f3256-2e55-4d7d-a84d-87de701d75fd)
![image](https://github.com/filamentphp/filament/assets/111913304/353b6958-362e-463f-b505-36040c4de199)
![image](https://github.com/filamentphp/filament/assets/111913304/fd67c91d-34e3-46d4-9278-baf0d4194802)

**Comment by pboivin** [comment]
What would this look like on your custom page?

**Reply by ramiropiro** [reply]
@pboivin they appear in the same row, I need them separately
![image](https://github.com/filamentphp/filament/assets/111913304/b3a957b8-9435-443c-931d-fafb89646207)

**Reply by pboivin** [reply]
Not sure if I understand correctly, maybe something like this?
```php
    TextColumn::make('_descripcion_corta')
        ->label('Carrera')
        ->formatStateUsing(fn ($record) =>
            new HtmlString(
                collect($record->carreras)->pluck('descripcion_corta')->join('<br>')
            )
        )
```

**Reply by ramiropiro** [reply]
@pboivin sory yes that puts one behind other but same row, I need to list them like a relation manager so I can click them separately and redirect to edit. like this image
![image](https://github.com/filamentphp/filament/assets/111913304/1c609bc3-2cd8-45f3-877b-9616fbfe4697)

**Reply by ramiropiro** [reply]
maybe I need a for each in the blade?

**Reply by pboivin** [reply]
Ah ok, I understand now :)
I think you'll need to adjust your table query... Instead of a `User` table, this should probably be a `Carrera` table?
```php
protected function getTableQuery()
{
    return auth()->user()->carreras();
}
```

**Reply by ramiropiro** [reply]
@pboivin thank you, It was the query, finally is working this way `auth()->user()->carreras->toQuery();`
